pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -575,7 +575,7 @@ func (rc *reconciler) updateStates(volumesNeedUpdate map[api.UniqueVolumeName]*r
 			continue
 		}
 		if volume.pluginIsAttachable {
-			err = rc.actualStateOfWorld.MarkDeviceAsMounted(volume.volumeName)
+			err = rc.actualStateOfWorld.MarkDeviceAsMounted(volume.volumeName, volume.devicePath)
```

Comment:
updateStates should be only triggered when kubelet restarts, reconciler needs to reconstruct the actual state of world (ASW). So it should be only called once after kubelet restarts. I think calling WaitForAttach here should be ok.